.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,103 @@
+name: Publish release
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  publish-release-package-cache:
+    runs-on: ubuntu-4-cores-150GB-ssd
+    container:
+      image: asterinas/asterinas:0.16.2-20251209
+      options: -v /dev:/dev --privileged
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build Asterinas NixOS patched packages and push to release cache
+        run: |
+          make kernel BOOT_PROTOCOL=linux-efi-handover64 || true
+          export CACHIX_AUTH_TOKEN=${{ secrets.CACHIX_AUTH_TOKEN_FOR_RELEASE_CACHE }}
+          make push_cachix USE_RELEASE_CACHE=1 2>&1 | tee cachix.log || true
+          tail --lines 10 cachix.log | grep -q -E "^(All done|Nothing to push)" || (echo "Push cachix failed" && exit 1)
```

Comment:
Why do we want to check the log? This is not a test. Shouldn't the exit code of `make push_cachix` already imply if the push succeeds or fails?